myriad-scheduler/src/main/java/org/apache/myriad/MyriadModule.java

```diff
@@ -168,4 +172,16 @@ ExecutorCommandLineGenerator providesCLIGenerator(MyriadConfiguration cfg) {
     }
     return cliGenerator;
   }
+  
+  protected MyriadConfiguration generateMyriadConfiguration(String configFile) {
+    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
Could be, will take a look and update accordingly

Sent from my iPhone

> On Jun 10, 2016, at 10:18 AM, Darin <notifications@github.com> wrote:
> 
> In myriad-scheduler/src/main/java/org/apache/myriad/MyriadModule.java:
> 
> > @@ -168,4 +172,16 @@ ExecutorCommandLineGenerator providesCLIGenerator(MyriadConfiguration cfg) {
> >      }
> >      return cliGenerator;
> >    }
> > +  
> > +  protected MyriadConfiguration generateMyriadConfiguration(String configFile) {
> > +    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
> I think a method like this exists in TaskUtils (which may not be the right place for it). Does it meet your needs?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>